csrankings-z.csv

```diff
@@ -212,6 +215,7 @@ Zhiyuan Liu 0001,Tsinghua University,http://nlp.csai.tsinghua.edu.cn/~lzy,dT0v5u
 Zhiyuan Luo 0001,Royal Holloway University of London,https://pure.royalholloway.ac.uk/portal/en/persons/zhiyuan-luo_cb78d990-b108-4090-8626-afe8c14f1936.html,NOSCHOLARPAGE
 Zhiyuan Wan,Zhejiang University,https://zhiyuan-wan.github.io,e-HfEUYAAAAJ
 Zhiyun Qian,Univ. of California - Riverside,http://www.cs.ucr.edu/~zhiyunq,NOSCHOLARPAGE
+Zhizheng Wu,CUHK-SZ,http://www.drwuz.com/,K6zhweAAAAAJ

```

Comment:
Web page says "Research scientist at Meta."

Reply:
Thanks for comments. Homepage of Prof. Wu has updated to CUHKSZ